pkg/operator/ceph/controller/predicate.go

```diff
@@ -409,6 +409,31 @@ func WatchControllerPredicate() predicate.Funcs {
 					return true
 				}
 				logger.Debugf("no change in CR %q", objNew.Name)
+
+			case *cephv1.CephBlockPoolRadosNamespace:
+				objNew := e.ObjectNew.(*cephv1.CephBlockPoolRadosNamespace)
+				logger.Debug("update event on CephBlockPoolRadosNamespace CR")
+				// If the labels "do_not_reconcile" is set on the object, let's not reconcile that request
+				IsDoNotReconcile := IsDoNotReconcile(objNew.GetLabels())
+				if IsDoNotReconcile {
+					logger.Debugf("object %q matched on update but %q label is set, doing nothing", objNew.Name, DoNotReconcileLabelName)
+					return false
+				}
+				diff := cmp.Diff(objOld.Spec, objNew.Spec, resourceQtyComparer)
+				if diff != "" {
+					logger.Infof("CR has changed for %q. diff=%s", objNew.Name, diff)
```

Comment:
The rook operator can operate on multiple resources in multiple namespaces. All info/debug messages must list the namespaced name so context is clear.